tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -264,3 +266,44 @@ def test_rhcloud_external_links():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.tier3
+def test_positive_generate_all_reports_job(target_sat):
+    """Generate all reports job via foreman-rake console:
+
+    :id: a9e4bfdb-6d7c-4f8c-ae57-a81442926dd8
+
+    :customerscenario: true
+
+    :Steps:
+        1. Execute Foreman GenerateAllReportsJob via cli.
+
+    :expectedresults: Reports generation works as expected.
+
+    :BZ: 2110163
+
+    :CaseAutomation: Automated
+
+    :CaseLevel: System
+    """
+    with target_sat.session.shell() as sh:
```

Comment:
Instead of running shell, you could also run single command like `'echo "command" | foreman-rake console'`

Reply:
Ya can go that way just taught of going with the shell approach!